docs/esterni/manuale_installazione/manuale_installazione.tex

```diff
@@ -0,0 +1,285 @@
+\begin{lstlisting}[language=bash]
+  $ sudo mysql -u <user> -p <password>
+\end{lstlisting}
+e sostituire:
+\begin{itemize}
+  \item \verb|<user>|: con l'user con il quale si desidera creare il database;
+  \item \verb|<password>|: con la password dell'user selezionato.
+\end{itemize}
+Effettuato il collegamento eseguire la seguente query per la creazione del database:
+\begin{verbatim}
+CREATE DATABASE OnlineStore;
+\end{verbatim}
+Successivamente creare le tabelle con le seguenti query:
+\begin{verbatim}
+USE OnlineStore;
+DROP TABLE PaymentEntries;
+DROP TABLE SettledPayments;
+DROP TABLE LastBlockSynced;
+CREATE TABLE PaymentEntries (id bigint, ecommerce varchar(255) not null,
+price bigint not null, primary key(id));
+CREATE TABLE SettledPayments (id bigint, item_id bigint not null,
+buyer varchar(255) not null, status int not null, created bigint not null,
+confirmed bigint, primary key(id));
+CREATE TABLE LastBlockSynced (id int(1), value bigint not null, primary key(id));
+INSERT INTO LastBlockSynced (id, value) VALUES (0, 0);
+\end{verbatim}
+\subsubsection{Metamask}
+Per poter utilizzare la webbapp è necessario metamask. Visitare \href{https://www.metamask.io}{metamask.io} per maggiori informazioni.\\\\
+Durante la creazione del wallet ricordare di salvare la mnemonic phrase poiché necessaria nei passi successivi.
+\paragraph{Configurazione network}\\
+Per l'inserimento e la configurazione del network Mumbai nel proprio wallet Metamask è necessario utilizzare i seguenti dati:
+\begin{itemize}
+\item Network Name: \textbf{Mumbai};
+\item New RPC Url: \textbf{https://rpc-mumbai.maticvigil.com/};
+\item ChainID: \textbf{80001};
+\item Currency Simbol: \textbf{MATIC};
+\item Block Explorer URL: \textbf{https://polygonscan.com/}.
+\end{itemize}
+Una volta inseriti i dati e aggiunto il network, ricordare di selezionare il network.
+\subsection{Configurazione}
+\label{sec:config_server}
+Una volta clonata la repository come spiegato nell'\hyperref[sec:intro]{introduzione}, è necessaria la creazione di un file `.env` dentro la cartella "Server". Il file deve essere strutturato come segue:
+\begin{verbatim}
+  PORT=8080
+  DB_HOST="localhost"
+  DB_USER="username"
+  DB_PWD="password"
+  DB_NAME="OnlineStore"
+  API_KEY=xxxxxxxxxxxxxxxxxxxxxxxx
+  SERVER_URL="tinyurl.com/xxxxxxxx"
+\end{verbatim}
+dove ciascun campo indica:
+\begin{itemize}
+  \item \textbf{PORT}: la porta che si desidera utilizzare;
+  \item \textbf{DB\_HOST}: l'host selezionato;
+  \item \textbf{DB\_USER}: l'user associato al database;
+  \item \textbf{DB\_PWD}: password dell' user utilizzato;
+  \item \textbf{DB\_NAME}: il nome del database da utilizzare, in questo caso "OnlineStore";
+  \item \textbf{API\_KEY}: le api key, ottenibili come spiegato al punto \hyperref[sec:provider]{2.1.2}.
+  \mbox{}
+  \item \textbf{SERVER\_URL}: strutturare il server url come segue:
```

Comment:
Dici strutturare SERVER_URL come segue e poi dici di fare un passo in più. Direi piuttosto, SERVER_URL: va creato tramite tinyurl un link che punta all'url nel formato ip.sslip.io o simili